app/assets/js/rtMedia.backbone.js

```diff
@@ -1689,7 +1689,11 @@ jQuery( document ).ready( function( $ ) {
 		$( this ).attr( 'disabled', 'disabled' );
 
 		// Sanitize comment content and escape html tags
-		comment_content_el.val( jQuery( '<span/>' ).text( jQuery.trim( comment_content_el.val() ) ).html() );
+		if( '' !== comment_content_el.val().replace(/\&nbsp;/g, '') ) {
+			comment_content_el.val( jQuery( '<span/>' ).text( jQuery.trim( comment_content_el.val() ) ).html() );
```

Comment:
Can't you use `comment_content_el.append()`?

`comment_content_el.append( jQuery( '<span/>' ).text( jQuery.trim( comment_content_el.val() ) ) );`

Reply:
Avoid `.html()` to get/set content.

Reply:
We can use append()  also but I think both the way is correct.
Example:-
input - 'hello'
output : &lt;span&gt;hello &lt;/span&gt;
According to my view .val() is better one because we just replacing value not appending a new within it.